mne/data/html_templates.py

```diff
@@ -100,3 +100,38 @@
     </tr>
 </table>
 """)
+
+evoked_template = Template("""
+<table class="table table-hover">
+    <tr>
+        <th>Content</th>
+        <td>{{evoked.comment}}</td>
+    </tr>
+    <tr>
+        <th>Data kind</th>
+        <td>{{evoked.kind}}</td>
+    </tr>
+    <tr>
+        <th>Data shape</th>
+        <td>{{f'{evoked.data.shape[0]} channels'}},
+        {{f'{evoked.data.shape[1]} time points'}}</td>
```

Comment:
```suggestion
        {{f'{evoked.data.shape[1]} samples'}}</td>
```

Reply:
I don't feel strongly about this suggestion, but to me it's good to be agnostic about whether the last dimension is actually "time" since users can put any data they want into our objects using e.g. `EvokedArray`